contract/src/orca.proposal.js

```diff
@@ -82,18 +111,19 @@ export const startOrcaContract = async (permittedPowers, config) => {
   const marshaller = await E(board).getPublishingMarshaller();
   console.log(marshaller);
 
-  /** @type {StartUpgradableOpts<OrcaSF>} */
+  /** @type {StartUpgradableOpts<OrcaSF>} **/
   const startOpts = {
     label: 'orca',
     installation,
     terms: undefined,
     privateArgs: {
-      storageNode,
-      marshaller,
+      localchain: await localchain,
+      // localchain: await E(localchain),
       orchestrationService: await cosmosInterchainService,
+      storageNode,
```

Comment:
any particular reason to move `storageNode` and `marshaller`? if not, let's put it back to avoid spurious diffs.